parachain/pallets/control/src/lib.rs

```diff
@@ -429,7 +429,7 @@ pub mod pallet {
 	impl<T: Config> Pallet<T> {
 		/// Send `command` to the Gateway on the channel identified by `origin`.
 		fn send(origin: ParaId, command: Command, pays_fee: PaysFee<T>) -> DispatchResult {
-			let message = Message { origin, command };
+			let message = Message { id: None, origin, command };
```

Comment:
If outbound queue is passed a messa with no ID, it will generate one for the message itself. See this commit: https://github.com/Snowfork/snowbridge/commit/b0e408e27129dff6fcd049dcfb3e7aa836141b46